app/helpers/form_helper.rb

```diff
@@ -442,6 +442,15 @@ def form_select_f(f, attr, array, select_options = {}, html_options = {})
     include_blank = select_options.delete(:include_blank)
     if include_blank
       addClass html_options, "include_blank"
+      blank_value = include_blank.is_a?(TrueClass) ? nil : include_blank
+      if array.kind_of?(Array) # incase array is options_for_select
+        array = array.to_a.dup
+        blank_option = [blank_value, nil]
+        array.insert(0, blank_option)
```

Comment:
personal preference, feel free to ignore
```suggestion
        array.unshift(blank_option)
```

Reply:
its a copy of 
https://github.com/theforeman/foreman/blob/8b1bd8f42907c10b97e5ae9d21d1306f85356729/app/helpers/form_helper.rb#L120-L121
so I'll rather leave it or change both?